### PR TITLE
Generates a company_token and passes it to the tracker.transistorsoft.com by default

### PR DESCRIPTION
**Pull request by ZeusTheTrueGod**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,95 +1,95 @@
...
+        "@angular/http": "4.1.2",
+        "@angular/platform-browser": "4.1.2",
+        "@angular/platform-browser-dynamic": "4.1.2",
+        "@ionic-native/core": "3.10.2",
+        "@ionic-native/device": "^3.12.1",
+        "@ionic-native/splash-screen": "3.10.2",
+        "@ionic-native/status-bar": "3.10.2",
+        "@ionic/storage": "2.0.1",
+        "cordova-android": "^6.2.3",
+        "cordova-background-geolocation-lt": "file:///Users/zeus/Documents/cordova-background-geolocation-lt",
```
Comment (christocracy) [inline package.json]: Have to be careful about committing references to your local file-system.

**Comment by christocracy** [comment]
Ok, go ahead and update yourself with my master.

**Comment by ZeusTheTrueGod** [comment]
Please check again

**Comment by christocracy** [comment]
Looks good.  Works on Android too.

**Pull request merged**